### DISCUSSION TITLE
Problema di autorizzazione?

### DISCUSSION THREAD

**Discussion by jacopogalli** [fattureincloud/api#179] [Q&A] [answered]
Buongiorno, ho un problema con un'applicazione privata e non riesco a capirne il motivo. 

Ho generato il token dando accesso completo a quello che mi interessa (clienti e fatture). 
Nella whitelist dell'applicazione c'è solo l'indirizzo email usato per accedere a FiC. 

Ho provato con Postman. 
/user/info funziona e ottengo il company_id
tutte le altre chiamate da qui in poi restituiscono sempre "Unauthorized.".

Sembra quasi che non tenga conto delle autorizzazioni ai moduli flaggati ... ho già provato a rigenerare svariate volte il token, cambiando le autorizzazioni ma il risultato non cambia. Non so cos'altro provare per risolvere il problema. 

Riuscite a darmi una mano?
Grazie

**Comment by valmoz** [comment, marked as answer]
Buongiorno @jacopogalli 
prima di tutto le chiederei di specificare quale metodo di autenticazione sta usando. Suppongo il token manuale.

Da quello che scrive, sembra che lei stia usando il metodo [Get User Info](https://developers.fattureincloud.it/api-reference#get-/user/info), mentre per recuperare il company id il metodo corretto è il [List User Companies](https://developers.fattureincloud.it/api-reference#get-/user/companies). Può trovare maggiori informazioni a questa pagina: [Company-scoped Methods](https://developers.fattureincloud.it/docs/basics/company-scoped-methods)

Grazie
Mauro

**Reply by valmoz** [reply]
Buongiorno @jacopogalli è riuscito a risolvere?

**Comment by jacopogalli** [comment]
Sì ho risolto, grazie.

Il lun 5 dic 2022, 14:31 Mauro Valota ***@***.***> ha scritto:
